sources/OpenMcdf.Extensions/OLEProperties/OLEPropertiesContainer.cs

```diff
@@ -126,8 +126,6 @@ internal OLEPropertiesContainer(CFStream cfStream)
                 UserDefinedProperties = new OLEPropertiesContainer(pStream.PropertySet1.PropertyContext.CodePage, ContainerType.UserDefinedProperties);
                 HasUserDefinedProperties = true;
 
-                UserDefinedProperties.ContainerType = ContainerType.UserDefinedProperties;
```

Comment:
The ContainerType property in  OLEPropertiesContaineris set to the value passed into the constructor, so I don't think it needs to be set here as well

Reply:
could remove this line without the other changes though